samples/interactive-tutorials/src/main/java/events/setup/EventsCreateGcsBucket.java

```diff
@@ -25,34 +25,30 @@
 import java.time.Instant;
 
 public class EventsCreateGcsBucket {
-
-  private static final String PROJECT_ID = ServiceOptions.getDefaultProjectId();
-
-  private static final Timestamp CURRENT_DATE =
-      Timestamp.newBuilder()
-          .setSeconds(Instant.now().getEpochSecond())
-          .setNanos(Instant.now().getNano())
-          .build();
-
-  private static final String BUCKET_NAME =
-      String.format("%s_events_%s", PROJECT_ID, CURRENT_DATE.getSeconds());
-
   public static void main(String... args) throws IOException {
```

Comment:
these have to be `String[]` to be proper main functions.

Reply:
Done.